src/Microsoft.ML.TimeSeries/RootCauseLocalizationType.cs

```diff
@@ -8,12 +8,19 @@
 
 namespace Microsoft.ML.TimeSeries
 {
+    public sealed class RootCauses
+    {
+        public double MeanGain { get; set; }
```

Comment:
Is this mean gain needed?  we could calcualte the mean gain according to gains in RootCause